src/redux/actions/listBookmarks.actions.js

```diff
@@ -0,0 +1,25 @@
+import ACTION_CONSTANTS from '../constants/constants';
+import { listBookmarksApi } from '../../services/api';
+
+const { LIST_BOOKMARKS, LIST_BOOKMARKS_FAILURE } = ACTION_CONSTANTS;
+
+const listSuccess = (data) => ({
+  type: LIST_BOOKMARKS,
+  payload: data,
+});
+
+const listFailure = (error) => ({
+  type: LIST_BOOKMARKS_FAILURE,
+  payload: error,
+});
+
+const listBookmarks = () => async (dispatch) => {
+  try {
+    const data = await listBookmarksApi();  
+    dispatch(listSuccess(data.data.articles));
+  } catch (error) {
+    dispatch(listFailure(error.response.data.message));
```

Comment:
Catch network errors, eg if there is no internet `error.response.data.message` will be undefined

Reply:
Okay working on that

Reply:
I have implemented that

Reply:
Cool.